src/steadystateproblem.cpp

```diff
@@ -191,9 +262,20 @@ bool SteadystateProblem::checkConvergence(const Solver *solver, Model *model) {
     return converged;
 }
 
+bool SteadystateProblem::checkSteadyStateSuccess() {
+    /* Did one of the attempts yield s steady state? */
+    if (steady_state_status[0] == SteadyStateStatus::success ||
```

Comment:
I think using `std::any_of` would be more idiomatic. Should be something like `std::any_of(steady_state_status.begin(), steady_state_status.end(), [](SteadyStateStatus status){return status == SteadyStateStatus::success;})`

May even be shorter but definitely better readability and easier to mantain if we decide to extend `steady_state_status`.